pandas/tests/scalar/timedelta/test_rendering.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+from pandas import Timedelta
```

Comment:
can you de-class these. I think we are calling this test_formats.py elsewhere?

Reply:
> I think we are calling this test_formats.py elsewhere?

In scalar.timestamps we're calling it test_rendering.  In tests.indexes we recently moved a bunch of thematically related tests to pre-existing test_formats.py files (that each only had one test in them, so seemed lonely).

I have a mild preference for "test_rendering" since it is pretty specifically about `__repr__` variants, whereas "test_formats" could reasonably refer to e.g. `to_csv`.

Reply:
just use test_formats for now.